### ISSUE TITLE
drag & drop "doesn't seem to match" seems too aggressive

### ISSUE THREAD

**Issue by gorgatron1** [syd711/vpin-studio#441]
**Describe the bug**

drag & drop "doesn't seem to match" seems too aggressive -- I selected `Futurama (Original 2024) Wizball 1.1` and dropped `Futurama (Original 2024) v1.2.1` on it to upgrade and got an alert telling me that it didn't seem to match

**To Reproduce**

1. have `Futurama (Original 2024) Wizball 1.1` in library
2. drag and drop `Futurama (Original 2024) v1.2.1.zip` on it to upgrade
3. get an alert about them not matching (see below)

I saw this in the logs -- that seems pretty close to me :-)

```
04-09 19:48:02.964 INFO  [JavaFX Application Thread] d.m.v.u.t.BaseDragDropHandler - Dropped file /Users/dkoski/Downloads/Futurama (Original 2024) v1.2.1.zip
04-09 19:48:08.771 INFO  [JavaFX Application Thread] d.m.v.c.u.StringSimilarity - Calculated similarity between 'Futurama (Original 2024) v1.2.1' and 'Futurama (Original 2024) Wizball 1.1' to 0.9362007168458781
```

The alert:

<img width="1252" alt="image" src="https://github.com/user-attachments/assets/62774165-5d22-4ad0-b60f-9b250c3afade">

**Expected behavior**

Expected: no alert as this is just a newer version of the same table

**Desktop (please complete the following information):**
 - OS: macOS 15

**Comment by syd711** [comment]
Yes, we already sharpened this match.

**Comment by syd711** [comment]
Fixed with 3.5.0
